src/tools/manifest2proto.ts

```diff
@@ -57,23 +58,69 @@ function encodePayload(payload: {}): Uint8Array {
 }
 
 async function particleSpecToProtoPayload(spec: ParticleSpec) {
+  let claims = [];
```

Comment:
Can you add a type for this? `let claims: ClaimProto[] = [];`

Reply:
Actually, I couldn't figure out how to import the generated classes. I made the following change in `manifest-proto.js`:
```
export const ClaimProto = RootNamespace.lookupType('arcs.ClaimProto');
```
Then, I changed the manfiest2proto.ts as follows:
```
import {ClaimProto} from "./manifest-proto.js"
...
...
let claims: ClaimProto[] = [];
```

However, I get the following error:
```
src/tools/manifest2proto.ts(61,15): error TS2749: 'ClaimProto' refers to a value, but is being used as a type here.
```

Any idea how to fix this?

Reply:
Ah right, it's a runtime value, not a class. TypeScript won't be able to know what the type is...

There's probably not a good type you can actually use here, except maybe a type like Message (if this js proto library has such a concept)